pylj/forcefields.py

```diff
@@ -29,11 +29,48 @@ def lennard_jones(dr, constants, force=False):
         The potential energy or force between the particles.
     """
     if force:
-        return 12 * constants[0] * np.power(dr, -13) - (
-            6 * constants[1] * np.power(dr, -7)
-        )
+        return 12 * constants[0] * np.power(dr, -13) -
```

Comment:
From a quick look at the travis build attempt it looks like there is an issue with the new line being immediately following the minus symbol.

Reply:
Multiline statements need a backslash at the end of incomplete lines, or to be wrapped in brackets (preferred style): https://stackoverflow.com/questions/53162/how-can-i-do-a-line-break-line-continuation-in-python

Reply:
On style, I am not fussed as the next time I push something to it I will be [black](https://github.com/python/black) formatting the whole thing. 
I will however add a comment about black formatting to the [CONTRIBUTING.md](https://github.com/arm61/pylj/blob/master/CONTRIBUTING.md) (see issue https://github.com/arm61/pylj/issues/44)

Reply:
I was referring to usual Python convention, not `pylj` style.